_maps/shuttles/escape_pod_default.dmm

```diff
@@ -0,0 +1,74 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE 
+"B" = (
+/turf/closed/wall/mineral/titanium,
+/area/shuttle/pod_1)
```

Comment:
Curious, are these assigned unique areas after being spawned in? If they're all set to pod_1 then they will all be operating off the same powernet (which wasn't the case prior, since each pod had/has a unique area.)

Reply:
Pods don't use power anyway but no they don't and I don't know of a clean way to handle that.

I'll have to think on that a bit.

Reply:
Shuttles aren't designed to handle multiple instances of the same area path. Nor are a lot of other things. The solution to this before was evidently to have many equivalent but differently named paths for the pods but to use that with templates would be very messy.

As it is it's not harmful. Since they don't use power anyway it can stay like this I think. Changing the code to allow multiple instances of the same area path is a possibility of the future, but not here.